src/components/Control.js

```diff
@@ -51,10 +51,10 @@ export const css = ({ isDisabled, isFocused }: State) => ({
 });
 
 const Control = (props: ControlProps) => {
-  const { children, cx, getStyles, isDisabled, isFocused, innerProps } = props;
+  const { children, cx, className, getStyles, isDisabled, isFocused, innerProps } = props;
   return (
     <Div
-      className={cx('control', { isDisabled, isFocused })}
+      className={(className ? `${className} ` : '') + cx('control', { isDisabled, isFocused })}
```

Comment:
let's simplify this by updating `cx` so it handles the className concatenation for us (may also be slightly more efficient..?)

after:

```js
className={cx('control', { isDisabled, isFocused }, className)}
```